db/schemas/schemas.py

```diff
@@ -11,7 +12,47 @@ class AssessmentRecordMetadata(SQLAlchemyAutoSchema):
 
     class Meta:
         model = AssessmentRecord
-        exclude = ["jsonb_blob", "application_json_md5"]
 
     workflow_status = Enum(Status)
     language = Enum(Language)
+
+
+ASSESSOR_TASK_LIST_METADATA_FIELDS = (
+    "short_id",
+    "project_name",
+    "workflow_status",
+    "jsonb_blob",
+    "fund_id",
+    "round_id",
+)
+
+
+class AssessorTaskListMetadata(AssessmentRecordMetadata):
+    """AssessorTaskListMetadata The marshmallow class used to turn SQLAlchemy
+    rows into json for return in http responses.
+
+    Fields are:
+    - project_name
+    - project_reference (short_id)
+    - workflow_status
+    - date_submitted
+    - fund_id
+    - round_id
+    """
+
+    class Meta:
+        model = AssessmentRecord
+        exclude = tuple(
+            field
+            for field in AssessmentRecord.__table__.columns.keys()
+            if field not in ASSESSOR_TASK_LIST_METADATA_FIELDS
+        )
```

Comment:
You can use `include` when you create the serialiser object - IE `serialiser = AssessmentRecordSchema(include=("short_id", "data_submitted"))`

Reply:
This way you don't have to write new objects for exclusion

Reply:
If we are missing a column add it and update the mapper here https://github.com/communitiesuk/funding-service-design-assessment-store/blob/main/db/queries/assessment_records/_helpers.py#L7

Reply:
I was using that originally but then at some point decided to go with this, I think it's when I (at least at the time) thought I had to write a new class and couldn't achieve what I was trying purely by passing properties (the `@post_dump` thing) - I'll take a look at re-using `include` if I can remove the `@post_dump` utilizing the docs you mentioned above

Reply:
Have changed to use `only` as an argument in the constructor, again if you know a better way please let me know, thanks for the feedback 😄